test/testvm.py

```diff
@@ -731,23 +749,37 @@ def _cleanup(self):
             print >> sys.stderr, "WARNING: Cleanup failed:", str(value)
 
     def kill(self):
+        # Attempt graceful shutdown first
+        if self._process and self._process.poll() is None:
+            self.execute(command="poweroff", ignore_ret_code=True)
```

Comment:
Why are we doing this. The whole point of the 'kill()' function is to kill the VM. We don't (or shouldn't) use this function if the VM is running in a way that would save disk state ... during tests the VM disks writes are discarded and we can just kill the VM.